coolkit/dependencies/dependency.py

```diff
@@ -69,42 +66,51 @@ def set_global_config():
     verify_file('~/.config/coolkit/global_config.py')
     shutil.copy(path_of_default_global_config, path)
 
-def install_dependencies(dependency_map, verbose = False):
-    supported_distros = _get_supported_distros(dependency_map)
-    distro = get_distro(supported_distros)
-    if(not distro and verbose):
-        print_err('unrecognised distro, please contact srbcheema2@gmail.com for full support for your distro',C.R)
-    elif(distro and verbose):
-        print_err('Distro detected to be '+distro+' based',C.G)
 
+def install_dependencies(dependency_map, verbose = False):
     all_installed = True
 
-    for d in dependency_map.keys():
-        if is_installed(d):
+    for dependency in dependency_map.keys():
+
+        if is_installed(dependency):
+            if verbose :
+                print_clr('.:Dependency '+dependency+' already installed...',C.G)
             continue
-        rules = dependency_map[d]
-        if distro and distro in rules.keys():
-            print_err('installing '+d+' dependency',C.G)
-            os.system(rules[distro])
-            if not is_installed(d):
-                print_err('please install ' +d+ ' dependency manually',C.Y)
-                print_err('try command : '+rules[distro],C.Y)
+
+        rules = dependency_map[dependency]
+        available_package_managers = _get_available_package_mangers( rules.keys(),verbose ) #dependency specific managers
```

Comment:
```suggestion
        available_package_managers = _get_available_package_mangers(rules.keys(), verbose)
```
for `()` keep rule like space only after `,` you may give space before `(` and after `)`.